### PR TITLE
Add changelog for 1.9.0 release

### PR DESCRIPTION
**Pull request by swissspidy**
See #8318

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -109,6 +109,18 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.9.0 =
+
+**Release Date:** July 13, 2021.
+
+* New feature: automated optimization of animated GIFs
+* New feature: Completely overhauled pre-publish checklist and improved checks.
```
Comment (dreamofabear) [inline readme.txt]:
```suggestion
* New feature: freshly redesigned checklist (now located next to the help center).
```

File: readme.txt [hunk 2]
```diff
@@ -109,6 +109,18 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
...
+* New feature: Optimize existing videos inserted via the WordPress media library.
+* New feature: Added image cropping when choosing publisher logo and poster image in the editor.
```
Comment (dreamofabear) [inline readme.txt]:
```suggestion
* New feature: crop publisher logo and poster images directly in the editor.
```

File: readme.txt [hunk 3]
```diff
@@ -109,6 +109,18 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.9.0 =
+
+**Release Date:** July 13, 2021.
+
+* New feature: automated optimization of animated GIFs
+* New feature: Completely overhauled pre-publish checklist and improved checks.
+* New feature: Optimize existing videos inserted via the WordPress media library.
```
Comment (dreamofabear) [inline readme.txt]:
```suggestion
* New feature: optimize existing videos in the WordPress media library.
```

File: readme.txt [hunk 4]
```diff
@@ -109,6 +109,18 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
...
+* New feature: Added image cropping when choosing publisher logo and poster image in the editor.
+* Added 9 brand new templates for creators to choose from.
```
Comment (dreamofabear) [inline readme.txt]:
```suggestion
* New feature: 9 new templates for creators to choose from.
```

File: readme.txt [hunk 5]
```diff
@@ -109,6 +109,18 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
...
+* Bumped minimum version requirements to PHP 7.0+ and WordPress 5.5+
+* Ensured compatibility with WordPress 5.8.
```
Comment (dreamofabear) [inline readme.txt]:
```suggestion
* Compatibility with WordPress 5.8.
```

File: readme.txt [hunk 6]
```diff
@@ -109,6 +109,18 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
...
+= 1.9.0 =
+
+**Release Date:** July 13, 2021.
+
+* New feature: automated optimization of animated GIFs
+* New feature: Completely overhauled pre-publish checklist and improved checks.
+* New feature: Optimize existing videos inserted via the WordPress media library.
+* New feature: Added image cropping when choosing publisher logo and poster image in the editor.
+* Added 9 brand new templates for creators to choose from.
+* Bumped minimum version requirements to PHP 7.0+ and WordPress 5.5+
```
Comment (dreamofabear) [inline readme.txt]:
```suggestion
* Minimum requirements are now PHP 7.0+ and WordPress 5.5+.
```

File: readme.txt [hunk 7]
```diff
@@ -109,6 +109,18 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.9.0 =
+
+**Release Date:** July 13, 2021.
+
+* New feature: automated optimization of animated GIFs
```
Comment (dreamofabear) [inline readme.txt]:
```suggestion
* New feature: animated GIFs are now automatically optimized as videos.
```

I'd also move this item below the cropping feature.

File: readme.txt [hunk 8]
```diff
@@ -225,6 +237,10 @@ For the plugin's full changelog, please see [the Releases page on GitHub](https:
 
 == Upgrade Notice ==
 
+= 1.9.0 =
+
+New templates, revamped pre-publish checklist, enhanced media optimization, bug fixes and performance improvements.
```
Comment (dreamofabear) [inline readme.txt]:
```suggestion
New templates, redesigned checklist, logo & poster image cropping, broader media optimization, bug fixes and performance improvements.
```

File: readme.txt [hunk 9]
```diff
@@ -109,6 +109,18 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
...
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.9.0 =
+
+**Release Date:** July 13, 2021.
+
+* New feature: animated GIFs are now automatically optimized as videos.
+* New feature: freshly redesigned checklist (now located next to the help center).
+* New feature: optimize existing videos in the WordPress media library.
+* New feature: crop publisher logo and poster images directly in the editor.
```
Comment (swissspidy) [inline readme.txt]:
```suggestion
* New feature: freshly redesigned checklist (now located next to the help center).
* New feature: optimize existing videos in the WordPress media library.
* New feature: crop publisher logo and poster images directly in the editor.
* New feature: animated GIFs are now automatically optimized as videos.
```

**Review by o-fernandez — APPROVED** [review]
LGTM

**Pull request merged**